config-templates/config-metarefresh.php

```diff
@@ -52,6 +52,12 @@
                         ],
                     ],
 
+                    'regex-template' => [
+                        "/^https:\/\/www\.example\.com\/sp$/" => [
```

Comment:
Probably this is much clearer because of not using `/` as the delimiter?

`"#^https://www\.example\.com/sp$#"`